src/code/audio_sound_params.c

```diff
@@ -1,238 +1,1290 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define SFX_PARAMS(importance, distParam, randParam, flags)                                 \
+    {                                                                                       \
+        importance, (((distParam) << SFX_PARAM_01_SHIFT) & SFX_PARAM_01_MASK) |             \
+                        (((randParam) << SFX_PARAM_67_SHIFT) & SFX_PARAM_67_MASK) | (flags) \
```

Comment:
```suggestion
        importance, ((((distParam) << SFX_PARAM_01_SHIFT) & SFX_PARAM_01_MASK) |          \
                     (((randParam) << SFX_PARAM_67_SHIFT) & SFX_PARAM_67_MASK) | (flags)) \
```
they call me the clang-format-11 trickster